code/game/objects/items/ego_weapons/teth.dm

```diff
@@ -125,3 +125,12 @@
 /obj/item/ego_weapon/regret/melee_attack_chain(mob/user, atom/target, params)
 	..()
 	user.changeNext_move(CLICK_CD_MELEE * 2) // Really Slow. This is the slowest teth we have, +0.4 to Eyes 1.6
+
+/obj/item/ego_weapon/cute
+	name = "SO CUTE!!!"
+	desc = "One may think, 'How can a weapon drawn from such a cute Abnormality be any good?' \
+		However, the claws are actually quite durable and sharp."
+	icon_state = "cute"
+	force = 15
```

Comment:
This is a bit too low damage for a teth, maybe try either 
attack speed 0.3 / force 8 
or force 22?

Reply:
Since wrist cutter's already 0.3
I'll settle at the middle aspd 0.5 / force 11

Reply:
Faster weapons have higher DPS in general actually. Wrist cutter is in a different class because it fits in a belt. 13/0.5 works too

Reply:
done